lstchain/onsite.py

```diff
@@ -275,10 +282,16 @@ def find_filter_wheels(run, database_url):
 
             filters = f"{w1:1d}{w2:1d}"
 
-    except Exception as e:
+    except Exception as e:  # In the case the entry says 'No available'
```

Comment:
except ValueError

Reply:
While checking this we ran into the DB server not being available (`pymongo.error.ServerSelectionTimeoutError`). This raised the point of leaving the bare `Exception` catching all issues (missing entry, value error, or server connection failure) and then raising `IOError` asking the user to specify the filter wheels. I do not want this to be a showstopper for the PR, so it's fine with me leaving the general `Exception`.